include/mega/node.h

```diff
@@ -291,6 +300,9 @@ struct MEGA_API LocalNode : public File
     // stored to rebuild tree after serialization => this must not be a pointer to parent->dbid
     int32_t parent_dbid = 0;
 
+    // whether this node can be synced to the remote tree
+    bool syncable = true;
```

Comment:
`syncable` and `mSyncable`?  perhaps these two could be distinguished a bit more - or should there only be one?

Reply:
what do you mean? syncable on `LocalNode` refers to syncing to the remote. syncable on `Node` refers to syncing to local.

Reply:
ah, I see - the gihub diff layout made it look like these two were in the same class

Reply:
The new member, anyway, should have been named `mSyncable` too, @cb-mega , don't you think so?